packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -256,7 +257,7 @@ export default class CodeSubmode extends Component<Signature> {
   private get fileIncompatibilityMessage() {
     if (this.isCard) {
       if (this.cardResource.cardError) {
-        return `Card preview failed. Make sure both the card instance data and card definition files have no syntax errors and that their data schema matches. `;
+        return `Card preview failed. Make sure both the card instance data and card definition files have no errors and that their data schema matches. `;
```

Comment:
I removed `syntax` because non-syntax errors can trigger this, like the use of an undefined `format(…` in the test.

Reply:
Is there any way we can highlight this? It’s not defined but you’d only see that trying to look at a card that adopts from it:

<img width="1077" alt="Boxel 2024-01-26 13-43-40" src="https://github.com/cardstack/boxel/assets/43280/eccba536-4a7c-4781-80fa-9b2198c96ef4">